grpcproxy/grpcproxy.go

```diff
@@ -0,0 +1,118 @@
+package grpcproxy
+
+import (
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+
+	"fmt"
+	"log"
+	"os"
+	"plugin"
+	"strings"
+
+	"github.com/akutz/gpds/lib"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+type conf struct{}
```

Comment:
this global name throws me off a bit - reading its declaration, I have no idea what it's for.